### PR TITLE
docs: add a Tilt debugging guide

### PR DESCRIPTION
**Pull request by nicks**
Hello @jazzdan, @l-korbes,

Please review the following commits I made in branch nicks/debug:

45e66fb8fb550aecf3e727989f0c0b9fb70590e9 (2020-01-03 16:46:02 -0500)
docs: add a Tilt debugging guide

### CODE REVIEW

File: docs/debug_faq.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+---
+title: Why is Tilt broken?
+layout: docs
+---
+
+Tilt is broken. What should you do?
+
+There are a lot of ways to get help, depending on your willingness to get your
```
Comment (jazzdan) [inline docs/debug_faq.md]: Depending on _how_ willing you are to get your hands dirty?

**Comment by nicks** [comment]
(note that i pretty shamelessly ripped off @jazzdan 's telemetry guide)

File: docs/debug_faq.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+```shell
+tilt dump webview
+```
+
+to see the complete state of the Tilt web UI.
+
+---
+
+### How can I monitor Tilt usage on my team?
```
Comment (jazzdan) [inline docs/debug_faq.md]: Do we want to document this?

File: docs/debug_faq.md (hunk 1 above)
Comment (nicks) [reply docs/debug_faq.md]: done!

File: docs/debug_faq.md (hunk 2 above)
Comment (nicks) [reply docs/debug_faq.md]: if people end up using it based on minimal documentation, that would be a great outcome!

**Pull request merged**